### ISSUE TITLE
Alert Modal on Android

### ISSUE THREAD

**Issue by jacksonrya** [FoodIsLifeBGP/banana-rn#67]
Currently, known in Android emulators, `TheAlertModal` pops up immediately after the app finishes building.

In some situations, it can be closed. In others, it can't!

**Comment by nexticodes** [comment]
Yeah I thought this was weird and I have been commenting out the Alert Modal just so I can login. Willing to investigate further!

**Comment by renburnett** [comment]
I have the same issue ):
